spec/lib/post_revisor_spec.rb

```diff
@@ -146,6 +146,76 @@
 
       expect { subject.revise!(admin, tags: ["new-tag"]) }.not_to change { Notification.count }
     end
+
+    it "doesn't create a small_action post when create_post_for_category_and_tag_changes is false" do
+      SiteSetting.create_post_for_category_and_tag_changes = false
+
+      expect { subject.revise!(admin, tags: ["new-tag"]) }.not_to change { Post.count }
+    end
+
+    describe "SiteSetting.create_post_for_category_and_tag_changes enabled" do
```

Comment:
```suggestion
    describe "when `create_post_for_category_and_tag_changes` site setting is enabled" do
```

Small suggestion since we're describing a scenario for the tests here.